transcom/mymove
# B 18065 int

[B-18065](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A870632)

## Summary

For INTEGRATION

This PR is for the PPM payment packet UI.

Sample PDF:
[ppm_payment_packet-2024-03-05T13_09_53.021Z.pdf](https://github.com/transcom/mymove/files/14496670/ppm_payment_packet-2024-03-05T13_09_53.021Z.pdf)

Note: PDF will also contain both bookmarks and footer/watermark.

Internal:

GET: /ppm-shipments/{ppmShipmentId}/payment-packet/
GHC/Office

GET: /ppm-shipments/{ppmShipmentId}/payment-packet/

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login into Milmove customer
2. on board PPM and submit
3. Login into Milmove office and review move
4. Approve PPM and "move details"
5. Login into Milmove customer and start uploading documentations
6. Add weights, pro-gear, expenses ..etc..
7. Log in to Milmove Office to approve customer payment
8. Verify payment packet download is available for Office user (see screenshot)
9. Log in to Milmove customer and verify payment packet download is available (see screenshot)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-03-15 at 10 01 48 AM](https://github.com/transcom/mymove/assets/162157577/ede84f79-d246-4de0-8372-6cbbbd38da9f)
![Screenshot 2024-03-13 at 6 27 45 PM](https://github.com/transcom/mymove/assets/162157577/193a3a00-365b-4ae7-bf3f-3ccd75925cce)
